docs/setup-hybrid-OVNKubernetes-cluster.md

```diff
@@ -19,54 +19,8 @@ $ sed -i 's/OpenShiftSDN/OVNKubernetes/g' <cluster_directory>/install-config.yam
 
 ## Create manifests
 
-Now generate the manifests for the previously created *install-config*:
-```sh
-$ openshift-install create manifests --dir=<cluster_directory>
-```
-This creates a `manifests` and `openshift` folder in your `<cluster_directory>`.
-Now create a `<cluster_directory>/manifests/cluster-network-03-config.yml` file with the following contents:
-```yml
-apiVersion: operator.openshift.io/v1
-kind: Network
-metadata:
-  name: cluster
-spec:
-  defaultNetwork:
-    ovnKubernetesConfig:
-      hybridOverlayConfig:
-        hybridClusterNetwork:
-        - cidr: 10.132.0.0/14
-          hostPrefix: 23
-```
-The above configuration is recommended for AWS and Azure clusters.
-
-### vSphere
-For vSphere clusters, you must add the `hybridOverlayVXLANPort` option to work around the pod-to-pod connectivity
-between hosts [issue](https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#pod-to-pod-connectivity-between-hosts-is-broken-on-my-kubernetes-cluster-running-on-vsphere):
-```yml
-apiVersion: operator.openshift.io/v1
-kind: Network
-metadata:
-  name: cluster
-spec:
-  defaultNetwork:
-    ovnKubernetesConfig:
-      hybridOverlayConfig:
-        hybridClusterNetwork:
-          - cidr: 10.132.0.0/14
-            hostPrefix: 23
-        hybridOverlayVXLANPort: 9898
-```
-#### Attention
-- The `hybridClusterNetwork` CIDR cannot overlap with the `clusterNetwork` CIDR
-- You cannot use Windows Server 2019 LTSC (1809) as it does not have the kernel feature required for using custom VXLAN 
-  ports
-
-## Create cluster
-
-Now proceed to cluster creation:
-```sh
-$ openshift-install create cluster --dir=<cluster_directory>
-```
+The manifests also must be changed in order to enable OVNKubernetes networking.
 
-Wait until cluster creation has succeeded and login credentials are shown.
+Follow the docs
+[here](https://docs.openshift.com/container-platform/4.12/networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.html)
```

Comment:
you can replace `4.12` with `latest` so we don't have to worry about this link going stale